README.md

```diff
@@ -96,7 +96,7 @@ If you have a Model such as `Friend` and you wish to upload friend's photo, File
 
 ```php
 // in Friend::init();
-$this->addField('file_id', [\Atk4\Filestore\Field\FileField::class]);
+$this->addField('photo', [\Atk4\Filestore\Field\FileField::class]);
```

Comment:
```suggestion
$this->addField('photo_file_id', [\Atk4\Filestore\Field\FileField::class]);
```

Reply:
It's not `id`, it's file `token`. Filestore do not link with other models trough `id` field, but using `token` field instead.
Why that was done like that I'm not exactly sure, but probably to completely decouple filestore tables from other tables in DB.

Reply:
then it should be `photo_file_token` ;-)

Reply:
ok :)

Reply:
and on line 109?